kernel/Kconfig

```diff
@@ -438,6 +438,12 @@ endmenu
 
 menu "Other Kernel Object Options"
 
+config MEM_SLAB_TRACE_MAX_UTILIZATION
+	bool "Enable getting maximum slab utiliazation."
```

Comment:
typo - "utiliazation"
Also the dot seems to be undesirable here.

Reply:
Fixed.